src/Core/src/Platform/Android/ButtonExtensions.cs

```diff
@@ -22,7 +22,24 @@ public static void UpdateTextColor(this AppCompatButton button, XColor color, Co
 
 		public static void UpdateTextColor(this AppCompatButton appCompatButton, IButton button, XColor defaultColor) =>
 			appCompatButton.SetTextColor(button.TextColor.Cleanse(defaultColor).ToNative());
-				
+
+		public static void UpdatePadding(this AppCompatButton appCompatButton, IButton button)
+		{
+			var context = appCompatButton.Context;
+
+			if (context == null)
+			{
+				return;
+			}
```

Comment:
@jsuarezruiz @PureWeen  not sure what the rules are for this type of thing... Should we be using the `IMauiContext`? Or is this no biggie because it can never be null?

Reply:
FYI I used the Label Padding implementation as a sample. So let me know if I should change it there as well

Reply:
I am just being dumb... Of course you want to use the context of the button... What else are you supposed to use? Another view? 🤦